components/cloud-event-gateway-proxy/README.md

```diff
@@ -0,0 +1,37 @@
+# Cloud Event Gateway Proxy
+
+## Overview
+
+The Cloud Event Gateway proxy receives Cloud Event publishing requests from the cluster workloads (microservice or serverless functions) and redirects it to the Enterprise Messaging Service Cloud Event Gateway.
+
+
+## Development
+
+Build
+
+```bash
+go mod vendor
+```
+
+Test
+
+```bash
+make test-local
+```
+
+Deploy
```

Comment:
I'd rather specify this as deploy w.r.t development image. and we should probably specify `ko` as a pre-requisite.

Reply:
It is already under a big header says `Development`.